src/components/KeyboardAvoidingView/index.ios.tsx

```diff
@@ -2,9 +2,9 @@
  * The KeyboardAvoidingView is only used on ios
  */
 import React from 'react';
-import {KeyboardAvoidingView as KeyboardAvoidingViewComponent} from 'react-native';
+import {KeyboardAvoidingView as KeyboardAvoidingViewComponent, KeyboardAvoidingViewProps} from 'react-native';
 
-function KeyboardAvoidingView(props) {
+function KeyboardAvoidingView(props: KeyboardAvoidingViewProps): React.ReactNode {
```

Comment:
I think we decided to not add return types to components

Reply:
Updated!